src/celeritas/Types.hh

```diff
@@ -29,6 +29,9 @@ namespace celeritas
 //! End-of-step (or perhaps someday within-step?) action to take
 using ActionId = OpaqueId<class ActionInterface>;
 
+//! Opaque index to IsotopeRecord in a vector
+using IsotopeId = OpaqueId<struct IsotopeRecord>;
+
```

Comment:
I think the IDs in this section are sorted by name, and the "detailed" IDs should be as well (but it looks like DetectorId needs to be moved)